framework/h5py_interface_creator.py

```diff
@@ -139,12 +140,10 @@ def __isGroup(self,name,obj):
     """
     if isinstance(obj,h5.Group):
       self.allGroupPaths.append(name)
-      self.raiseAMessage('Accessing group named ' +name)
-      if "EndGroup" in obj.attrs:
-        self.allGroupEnds[name]  = obj.attrs["EndGroup"]
-      else:
-        self.raiseAWarning('not found attribute EndGroup in group ' + name + '.Set True.')
+      try   : self.allGroupEnds[name]  = obj.attrs["EndGroup"]
+      except:
```

Comment:
Bare excepts are not allowed by the code standard.  This should catch a specifc exception or should get the execption and report it.